public/stylesheets/main.css

```diff
@@ -0,0 +1,141 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Texturina:wght@100&display=swap');
+
+body {
+  font-family: 'Texturina', serif;
+}
+
+/* .container1, .container2, .container3 {
+  width: fit-content;
+  margin: auto;
+} */
```

Comment:
Please remove these commented out classes.

Reply:
this one has been deleted, currently working on other requests